subprojects/packagefiles/sfml/meson.build

```diff
@@ -6,22 +6,20 @@ project(
     'cpp_std=c++17',
     'wrap_mode=forcefallback',
   ],
-  meson_version: '>=0.49.0',
+  meson_version: '>=0.59.0',
 )
 
 cpp = meson.get_compiler('cpp')
 
-openal_dep = dependency('openal')
-thread_dep = dependency('threads')
+graphics_opt = get_option('graphics')
+audio_opt = get_option('audio')
+system_opt = get_option('system')
+window_opt = get_option('window')
+network_opt = get_option('network')
```

Comment:
Looking at the cmake build system, `system` does not seem to be optional:
```cmake
# CMakeLists.txt

sfml_set_option(SFML_BUILD_WINDOW TRUE BOOL "TRUE to build SFML's Window module. This setting is ignored, if the graphics module is built.")
sfml_set_option(SFML_BUILD_GRAPHICS TRUE BOOL "TRUE to build SFML's Graphics module.")
sfml_set_option(SFML_BUILD_AUDIO TRUE BOOL "TRUE to build SFML's Audio module.")
sfml_set_option(SFML_BUILD_NETWORK TRUE BOOL "TRUE to build SFML's Network module.")
```
```cmake
# src/SFML/CMakeLists.txt

# sfml-system
add_subdirectory(System)

# sfml-main and sfml-activity
if(SFML_OS_WINDOWS OR SFML_OS_ANDROID OR SFML_OS_IOS)
    add_subdirectory(Main)
endif()

# sfml-window
if(SFML_BUILD_WINDOW OR SFML_BUILD_GRAPHICS)
    add_subdirectory(Window)
endif()

# sfml-network
if(SFML_BUILD_NETWORK)
    add_subdirectory(Network)
endif()

# sfml-graphics
if(SFML_BUILD_GRAPHICS)
    add_subdirectory(Graphics)
endif()

# sfml-audio
if(SFML_BUILD_AUDIO)
    add_subdirectory(Audio)
endif()

```

Reply:
fix by 0b400456a2108332c2a3b7ce4cb21a6293408709